x/bank/internal/keeper/keeper_test.go

```diff
@@ -140,8 +95,12 @@ func TestKeeper(t *testing.T) {
 func TestSendKeeper(t *testing.T) {
 	input := setupTestInput()
 	ctx := input.ctx
+
+	blacklistedAddrs := make(map[string]bool)
+	blacklistedAddrs[sdk.AccAddress([]byte("moduleAcc")).String()] = true
```

Comment:
This seems like it can be removed and in `common_test`, I don't see any usage of it?